library/cohesity_job.py

```diff
@@ -381,17 +381,46 @@ def parse_vmware_protection_sources_json(response, vm_names):
     return list(set(ids))
 
 
+def get_tag_ids(module, tags, parentSourceId, token):
+    server = module.params.get('cluster')
+    validate_certs = module.params.get('validate_certs')
+    #token = job_details['token']
+    try:
+        uri = "https://" + server + "/irisservices/api/v1/public/protectionSources?id=%s&excludeTypes=kResourcePool,kVirtualMachine,kComputeResource,kDatastore" % str(parentSourceId)
```

Comment:
truncate line , also please use sdk for this.

Reply:
Ashish, other requests in the module are made using direct API call. Can i use SDK only for this call?